artiq/frontend/artiq_coremgmt.py

```diff
@@ -73,6 +73,14 @@ def get_argparser():
                          help="key and file whose content to be written to "
                               "core device config")
 
+    subparsers.add_parser("write_ch_name",
+                          help="store channel number and corresponding device "
+                               "name from device_db.py to core device config")
```

Comment:
It's called the device database. ``device_db.py`` is just the default file name.